CHANGELOG.md

```diff
@@ -2,6 +2,71 @@
 
 ## Unreleased
 
+### API
+
+* Update Span javadoc to allow null/empty attr values
+  ([#5616](https://github.com/open-telemetry/opentelemetry-java/pull/5616))
+
+### SDK
+
+* Add Sdk{Signal}ProviderBuilder#addResource(Resource) method to merge Resource
+  with current
+  ([#5619](https://github.com/open-telemetry/opentelemetry-java/pull/5619))
+
+#### Metrics
+
+* Add LongHistogramAdviceConfigurer to improve api surface types
+  ([#5689](https://github.com/open-telemetry/opentelemetry-java/pull/5689))
+
```

Comment:
```suggestion
* Increase metric name maximum length from 63 to 255 characters
  ([#5697](https://github.com/open-telemetry/opentelemetry-java/pull/5697))

```

Reply:
We're going to be holding off on [#5697](https://github.com/open-telemetry/opentelemetry-java/pull/5697#issuecomment-1673753956) until next release.